spec/features/sign_in_to_Chitter_spec.rb

```diff
@@ -0,0 +1,15 @@
+feature 'User sign in' do
+  let!(:user) do
+    User.create(name: 'Ezzy Elliott',
+              username: 'ezzye',
+              email: 'ezzy.elliott@gmail.com',
+              password: '123456',
+              password_confirmation: '123456')
+    end
```

Comment:
Lint/BlockAlignment: `end` at 8, 4 is not aligned with `let!(:user) do` at 2, 2.